python-sdk/src/astro/table.py

```diff
@@ -12,7 +13,7 @@
 from astro.settings import OPENLINEAGE_EMIT_TEMP_TABLE_EVENT
 
 MAX_TABLE_NAME_LENGTH = 62
-TEMP_PREFIX = "_tmp_"
+TEMP_PREFIX = "_tmp"
```

Comment:
Why did we changed the prefix?

Reply:
It was inconsistent before so we had to just between the above and this https://github.com/astronomer/astro-sdk/pull/1237#discussion_r1020094281